src/containers/Flow/Flow.test.tsx

```diff
@@ -53,13 +60,22 @@ it('should support keywords in a separate language', async () => {
 });
 
 it('should not allow special characters in keywords', async () => {
-  const { container, getByText } = render(flow({ params: {} }));
+  const { container, getByText, findByTestId } = render(flow({ params: {} }));
 
-  await waitFor(() => {});
-  fireEvent.change(container.querySelector('input[name="name"]'), {
+  const nameInput = await (await findByTestId('formLayout')).querySelector('input[name="name"]');
```

Comment:
Can we check a better way to achieve this? The code looks bit funky :)

Reply:
@kurund I referred it directly from docs, so I used it. Ok, let me check for other ways.

Reply:
@enigmaticmahesh I was referring to await before await which you have removed now. So it seems fine.